README.md

```diff
@@ -3194,6 +3195,63 @@ You can configure eventual consistency with the `ClientConfig.properties` below:
 
 - `hazelcast.invalidation.reconciliation.interval.seconds`: Default value is `60` seconds. This is a periodic task that scans cluster members periodically to compare generated invalidation events with the received ones from the client Near Cache.
 
+### 7.8.3. Pipelining
+
+With the pipelining, you can send multiple parallel requests with a defined concurrency limit without waiting for their results.
```

Comment:
No need to use the article ("the") for the act of pipelining throughout this content.